tests/issues/issue-340/issue-339.test.ts

```diff
@@ -0,0 +1,14 @@
+import { compareFiles } from 'tests/common';
```

Comment:
The folder name issue-340 is misleading, because it has nothing todo with #340 
Also there is a file `issue-339` in issue folder `340`?

Please reference the correct issue (told you to open a new issue for that case, so we can address it. Please do not necromant a already closed/merge PR)